Dashboards/dashboard-Home-old.json

```diff
@@ -0,0 +1,275 @@
+{
+	"id": "my-home-dashboard",
```

Comment:
mistake?

Reply:
No, should have the same ID as the other one, since they are targeted for different versions